numberOfDisparities: 80
minDisparity: 0
SADWindowSize: 3
preFilterCap: 63
P1: 768
P2: 1536
uniquenessRatio: 15
speckleWindowSize: 50
speckleRange: 16
disp12MaxDiff: 1
fullDP: true
<<<<<<< HEAD
ignore_border_left: 30
=======
ignore_border_left: 40
>>>>>>> 970e495b054521a1038aed9ce91708c32e2cb9d9

#numberOfDisparities: 80
#minDisparity: 0
#SADWindowSize: 5
#preFilterCap: 0
#P1: 0
#P2: 0
#uniquenessRatio: 0
#speckleWindowSize: 0
#speckleRange: 0
#disp12MaxDiff: 0
#fullDP: false
